src/main/kotlin/com/vk/admstorm/ssh/YubikeyHandler.kt

```diff
@@ -44,9 +44,10 @@ class YubikeyHandler {
             PasswordSafe.instance.getPassword(credentialAttributes)!!
         }
 
-        val killOutput = CommandRunner.runLocally(project, "pkill ssh-agen")
-        if (killOutput.exitCode != 0) {
-            LOG.warn("pkill ssh-agen exited with non-zero code while Yubikey reset")
+        val openscPath = if (SystemInfo.isLinux) {
```

Comment:
Why move it?

Reply:
As for me, it’s more beautiful when the variables that are auxiliary for calculating business logic are located higher than it, especially since there is branching